libs/components/router/testing/src/href/href-harness.ts

```diff
@@ -0,0 +1,42 @@
+import { HarnessPredicate } from '@angular/cdk/testing';
+import { SkyComponentHarness } from '@skyux/core/testing';
+
+import { SkyHrefHarnessFilters } from './href-harness-filters';
+
+export class SkyHrefHarness extends SkyComponentHarness {
+  static hostSelector = '[skyHref]';
+
+  /**
+   * Gets a `HarnessPredicate` that can be used to search for a
+   * `SkyHrefHarness` that meets certain criteria.
+   */
+  public static with(
+    filters: SkyHrefHarnessFilters
+  ): HarnessPredicate<SkyHrefHarness> {
+    return this.getDataSkyIdPredicate(filters);
+  }
+
+  async getHref(): Promise<string | null> {
+    return this.forceStabilize().then(async () => {
+      return await (await this.host()).getAttribute('href');
+    });
+  }
+
+  async getLinkText(): Promise<string> {
```

Comment:
What do you think about renaming this to `getText` since it might be getting the "unlinked" text if the user doesn't have access?

Reply:
Switched the name to `getText` and updated the logic to check visibility so it returns the effective text.